tests/tests/control_csawmg

```diff
@@ -24,7 +24,7 @@ export SYEAR=2021
 export SMONTH=03
 export SDAY=22
 export SHOUR=06
-export RESTART_INTERVAL=12
+export RESTART_INTERVAL=0
```

Comment:
Just to double check: does this mean the restart files will be written after the first time step?